scripts/create-conda-recipe.py

```diff
@@ -0,0 +1,188 @@
+import re
+from collections import defaultdict
+from copy import deepcopy
+
+import requests
+import yaml
+from bs4 import BeautifulSoup
+
+
+def get_extra_requires(path):
+    try:
+        with open(path) as fp:
+            extra_deps = defaultdict(set)
+            for k in fp:
+                if k.strip() and not k.startswith('#'):
+                    tags = set()
+                    if ':' in k:
+                        k, v = k.split(':')
+                        tags.update(vv.strip() for vv in v.split(','))
+
+                    k = re.split('([<>=])', k, 1)
+
+                    # If we have some version requrement, it will be returned as a
+                    # separate item by the split function
+                    if len(k) == 3:
+                        k = [k[0], k[1] + k[2]]
+                    assert len(k) <= 2, 'requirement should not have more than 2 parts'
+
+                    # Can not use extra requirements in conda - i.e. uvicorn[standard]
+                    k[0] = re.sub(r'\[\w+\]', '', k[0])
+
+                    # The docker package is docker-py on conda-forge
+                    if k[0] == 'docker':
+                        k[0] = 'docker-py'
+
+                    # Pytorch package is pytorch-cpu (gpu not needed for demo) on forge
+                    if k[0] == 'torch':
+                        k[0] = 'pytorch-cpu'
```

Comment:
How about something along the lines of:
```
    k[0] = {'docker': 'docker-py', 'torch': 'pytorch-cpu'}.get(k[0], k[0])
```
This way you only have to extend the dictionary instead of having to modify the  if checks every time?